src/schema/v2/artistSeries.ts

```diff
@@ -0,0 +1,198 @@
+          },
+          size: {
+            type: GraphQLInt,
+          },
+        },
+        resolve: ({ artist_ids }, args, { artistsLoader }) => {
+          if (artist_ids.length === 0) return []
+
+          const gravityArgs = {
+            ids: artist_ids,
+            ...args,
+          }
+
+          return artistsLoader(gravityArgs).then(({ body }) => body)
+        },
+      },
+      description: {
+        type: GraphQLString,
+      },
+      descriptionFormatted: markdown(({ description }) => description),
+      published: {
+        type: new GraphQLNonNull(GraphQLBoolean),
+      },
+      featured: {
+        type: new GraphQLNonNull(GraphQLBoolean),
+      },
+      representativeArtworkID: {
+        type: GraphQLID,
+        resolve: ({ representative_artwork_id }) => representative_artwork_id,
+      },
+      forSaleArtworksCount: {
+        type: new GraphQLNonNull(GraphQLInt),
+        resolve: ({ for_sale_artworks_count }) => for_sale_artworks_count,
+      },
+      artworksCount: {
+        type: new GraphQLNonNull(GraphQLInt),
+        resolve: ({ artworks_count }) => artworks_count,
+      },
+      artworksCountMessage: {
+        type: GraphQLString,
+        resolve: ({ artworks_count, for_sale_artworks_count }) => {
+          let artworksCountMessage
+
+          if (for_sale_artworks_count) {
+            artworksCountMessage = `${for_sale_artworks_count} available`
+          } else {
+            artworksCountMessage = `${artworks_count} ${
+              artworks_count === 1 ? "work" : "works"
+            }`
+          }
+
+          return artworksCountMessage
+        },
+      },
+      image: {
+        type: Image.type,
+        resolve: async (
+          { image_url, image_height, image_width, representative_artwork_id },
+          _args,
+          { artworkLoader }
+        ) => {
+          if (image_url) {
+            return {
+              image_url,
+              original_width: image_width,
+              original_height: image_height,
+            }
+          } else if (representative_artwork_id) {
+            const { images } = await artworkLoader(representative_artwork_id)
+            return normalizeImageData(getDefault(images))
+          }
+        },
+      },
+      filterArtworksConnection: filterArtworksConnectionWithParams(
+        ({ _id }) => {
+          return {
+            artist_series_id: _id,
+          }
+        }
+      ),
+    }
+  },
+})
+
+export const ArtistSeries: GraphQLFieldConfig<void, ResolverContext> = {
+  type: ArtistSeriesType,
+  args: {
+    id: {
+      type: new GraphQLNonNull(GraphQLID),
+    },
+  },
+  resolve: (_root, { id }, { artistSeriesLoader }) => {
+    return artistSeriesLoader(id)
+  },
+}
+
+export default ArtistSeries
+
+export const ArtistSeriesConnectionType = connectionWithCursorInfo({
+  nodeType: ArtistSeriesType,
+  connectionFields: {
+    totalCount: {
+      type: new GraphQLNonNull(GraphQLInt),
```

Comment:
This was technically non-null in the stitched schema, and while there looks to be some null-safe references to it (like https://github.com/artsy/eigen/blob/83f9134e83195aae6bf6f732f5b13f171822124b/src/app/Scenes/ArtistSeries/ArtistSeriesMoreSeries.tsx#L40 ), making this field nullable is a **breaking** schema change, so a bit silly but might as well do it this way